src/main/java/application/baseballGame/BaseballGame.java

```diff
@@ -0,0 +1,17 @@
+package application.baseballGame;
+
+public class BaseballGame {
+    static final int NUMBER_COUNT_RULE = 3;
```

Comment:
```suggestion
    public static final int NUMBER_COUNT_RULE = 3;
```

요 상수는 Computer, User, GameResult에서 사용되는 것으로 보여집니다.

특정 객체에서만 사용되는 필드라면 관련 클래스에 정의되어 있는 것이 좋을 것 같고, 
현재 도메인과 관련된 클래스에서만 사용되는 상수라면 enum으로 정의한 뒤에 전역에서 사용할 수 있도록 제공하는 방법도 있을 것 같습니다.